CHANGELOG.md

```diff
@@ -2512,9 +2512,11 @@ Swift 2.2
     AnyObject>`.  It bridges in O(1), without memory allocation.
 
   - `Dictionary<K, V>` has an implicit conversion to `NSDictionary`.
-    `Dictionary<K, V>` bridges to `NSDictionary` iff both `K` and `V` are
+    `Dictionary<K, V>` bridges to `NSDictionary` [iff] both `K` and `V` are
```

Comment:
It makes it clearer (to me) that it's a link rather than some kind of placeholder, or an array that ought to be in backticks. But maybe I've just used Markdown long enough to have habits and preferences, which doesn't automatically make them good ones.